generator.go

```diff
@@ -139,6 +161,94 @@ func (p *property) readFromStruct(t reflect.Type) {
 	}
 }
 
+func (p *property) addValidatorsFromTags(tag *reflect.StructTag) {
+	switch p.Type {
+	case "string":
+		p.addStringValidators(tag)
+	case "number", "integer":
+		p.addNumberValidators(tag)
+	}
+}
+
+// Some helper functions for not having to create temp variables all over the place
+func int64ptr(i interface{}) *int64 {
+	j := reflect.ValueOf(i).Convert(reflect.TypeOf(int64(0))).Interface().(int64)
```

Comment:
What would you want me to do if the interface is not convertible? Just panic? The generator currently doesn't have a mechanism for returning errors, and in order to add that, I'd have to change the signature of every method so that it can return an error.